src/changelog.ts

```diff
@@ -43,7 +44,10 @@ const fileOptions = [
   'history.md',
   'HISTORY',
   'history',
+  'RELEASES.md',
+  'RELEASES',
 ];
+const fileRegex = /changelog|changes|history|release/i;
```

Comment:
will this have a false positive for a name like `generateChangelog.js` at root?

Reply:
indeed, I'll add an extension check